books/free-programming-books-langs.md

```diff
@@ -1156,7 +1156,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [JS Robots](https://web.archive.org/web/20201029045339/http://markdaggett.com/images/ExpertJavaScript-ch6.pdf) - Mark Daggett (PDF) *(:card_file_box: archived)*
 * [Leaflet Tips and Tricks: Interactive Maps Made Easy](https://leanpub.com/leaflet-tips-and-tricks/read) - Malcolm Maclean (HTML)
 * [Learning JavaScript Design Patterns](http://addyosmani.com/resources/essentialjsdesignpatterns/book/) - Addy Osmani (HTML)
-* [Let's Learn ES6](https://bubblin.io/cover/let-s-learn-es6-by-ryan-christiani#frontmatter) - Ryan Christiani (Superbook format)
+* [Let's Learn ES6](https://bubblin.io/book/let-s-learn-es6-by-ryan-christiani#frontmatter) - Ryan Christiani (Superbook format)
```

Comment:
this didn't work for me, will try later

Reply:
Huh, you can't? I just triple-check, and the link takes me to the book. Here's the link: [https://bubblin.io/book/let-s-learn-es6-by-ryan-christiani#frontmatter](https://bubblin.io/book/let-s-learn-es6-by-ryan-christiani#frontmatter). If you can't access it, I wonder if it's due to some regional restriction.